src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -182,7 +182,10 @@ private static bool IsRedHatFamilyAndVersion(int major = -1, int minor = -1, int
         /// Get whether the OS platform matches the given Linux distro and optional version.
         /// </summary>
         /// <param name="distroId">The distribution id.</param>
-        /// <param name="versionId">The distro version.  If omitted, compares the distro only.</param>
+        /// <param name="major">The distro major version.  If omitted, compares the distro only.</param>
+        /// <param name="minor">The distro minor version.  If omitted, compares the distro only.</param>
+        /// <param name="build">The distro build version.  If omitted, compares the distro only.</param>
+        /// <param name="revision">The distro revision version.  If omitted, compares the distro only.</param>
```

Comment:
📝 These comments are not precise. It should say something like this:

> If omitted, this portion of the version is not included in the comparison.

The wording may vary, but the key is excluding a parameter impacts only that field and not the entire version number as the current comment suggests.